portable/RVDS/ARM_CM4F/port.c

```diff
@@ -468,36 +468,36 @@ __asm void xPortPendSVHandler( void )
     /* Is the task using the FPU context?  If so, push high vfp registers. */
     tst r14, #0x10
     it eq
-    vstmdbeq r0!, {s16-s31}
+    vstmdbeq r0 !, { s16 - s31 }

```

Comment:
Likewise "r0!" was correct, it specifies writing back to the register.  Likewise for all similar edits.